src/Compilers/CSharp/Portable/Symbols/FieldSymbol.cs

```diff
@@ -373,6 +373,30 @@ public sealed override bool HasUnsupportedMetadata
 
         #endregion
 
+        /// <summary>
+        /// Is this a field of a tuple type?
+        /// </summary>
```

Comment:
There is no distinction, that is why there is no comment.

---
In reply to: [61811380](https://github.com/dotnet/roslyn/pull/11006#discussion_r61811380) [](ancestors = 61811380)